src/features/identityCheck/components/layout/PageWithHeader.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Text } from 'react-native'
+
+import { PageWithHeader } from 'features/identityCheck/components/layout/PageWithHeader'
+import { mockGoBack } from 'features/navigation/__mocks__/useGoBack'
+import { render } from 'tests/utils'
+
+describe('<PageWithHeader/>', () => {
+  it('should render correctly', () => {
```

Comment:
selon moi, le snapshot testing est là pour figer un résultat, il ne valide pas une règle métier (surtout lorsque le composant n'a pas de logique métier)

je pense que nommer un test de snapshot `should render correctly` serait aussi pertinent que de le nommer `should work`, ou de ne pas le nommer

je pense qu'il pourrait etre plus intéressant résumé le cas testé (surtout lorsqu'on a différent cas de tests avec différents snapshots)

dans ce cas, je propose

```suggestion
  test('nominal case', () => {
```

Reply:
Je pense que ce snapshot et ce teste n'est finalement pas utile. Je vais le supprimer !

Reply:
pourquoi penses-tu qu'il n'est pas utile ?

Reply:
Parce que je ne pense pas que ce soit utile de tester la condition d'affichage, alors que j'utilise ce composant sur beaucoup de pages, qui elles mêmes on leurs snapshots. Si le composant `PageWhitHeader` déconne, je vais le voir dans les snapshots des autres pages.

Reply:
Après je peux le laisser, ça ne coute rien. Mais changer le titre du teste je ne suis pas pour. Car tous les snapshots de l'app on ce titre de teste. Il faudrait changer tous les autres tests pour être aligné.